src/shared/locale/en.json

```diff
@@ -834,6 +834,9 @@
     "title": "Send",
     "wallet": "Wallet"
   },
+  "walletDetails": {
+    "simpleTitle": "Wallet details"
```

Comment:
maybe better to create group with common translations and groups with specific for each wallet type?